src/js/tracing/reactnativetracing.ts

```diff
@@ -454,7 +454,10 @@ export class ReactNativeTracing implements Integration {
           },
         });
       }
-
+      scope.setContext('app', { view_names: [context.name] });
```

Comment:
We should check if `app` exists in the scope, so we don't overwrite other keys set by a user, only the `view_names`.